### DISCUSSION TITLE
Excluded network subnets

### DISCUSSION THREAD

**Discussion by rebel313** [bb-Ricardo/netbox-sync#252] [Q&A] [answered]
Hi,

First of all this is great tool.
Can we have **excluded_subnets** similar to permitted_subnets?

The thing is that I want all subnets with permitted_subnets = 0.0.0.0/0, but I have one subnet that is cloned from production to test VM and the IP address space overlaps. This leads to changing VM names between test and prod environment and treating them as one when syncing.

Thanks!

**Comment by bb-Ricardo** [comment, marked as answer]
hi, this is implemented already but wrongly documented (will fix that in the next release):

https://github.com/bb-Ricardo/netbox-sync/blob/7db3624f90484e15a03911112640763f73129781/settings-example.ini#L327-L331

**Reply by rebel313** [reply]
Thanks, I was looking at vmware and missed this redfish config.

**Reply by bb-Ricardo** [reply]
fixed in current `development` branch.

**Comment by rebel313** [comment]
BTW, small typo https://github.com/bb-Ricardo/netbox-sync/blob/7db3624f90484e15a03911112640763f73129781/settings-example.ini#L149 `s/*./.*`

**Reply by bb-Ricardo** [reply]
Think you. fixed in current `development` branch.
